docs/organizations/overview.mdx

```diff
@@ -74,19 +74,15 @@ Membership requests are requests from users who want to join an organization. A
 
 ## Active organization
 
-There are two types of accounts that a single user can have in a Clerk application: a personal account and an organization account. A personal account is the default account type, and it is created when a user signs up for your application. An organization account is created when a user creates or joins an organization within your application.
+When a user is a member of an organization, they can switch between their personal workspace and an organization workspace. The organization workspace that a user is currently viewing is called the **active organization**. The active organization determines which organization-specific data the user can access and which roles and permissions they have within the organization.
```

Comment:
```suggestion
When a user is a member of an organization, they can switch between their personal workspace and an organization workspace. The organization workspace that a user is currently viewing is called the **active organization**. The active organization determines which organization-specific data the user can access and which role and related permissions they have within the organization.
```
A user can only have one role, and permissions are tied to that, so just tightening up that part a bit.